### PR TITLE
vcsim: add esxcli support

### PR DESCRIPTION
**Pull request by dougm**
api: move cli/host/esxcli.Command to cli/esx package

BREAKING: Package "github.com/vmware/govmomi/govc/host/esxcli" is now "github.com/vmware/govmomi/cli/esx"
A 'context.Context' param has also been added to the package's methods.
See cli/esx/example_test.go for usage.

### CODE REVIEW

File: cli/esx/command.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
```
Comment (tenthirtyam) [inline cli/esx/command.go:1]: Suggestion to update the headers to the following but can be left to a seperate PR.

THis has been done on several other projects based on discussion with Chip C.

```go
// © Broadcom. All Rights Reserved.
// The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
// SPDX-License-Identifier: Apache-2.0
```

**Comment by dougm** [comment]
Thanks @akutz !

File: cli/esx/command.go (hunk 1 above)
Comment (dougm) [reply cli/esx/command.go:1]: Thanks @tenthirtyam

Comment (tenthirtyam) [reply cli/esx/command.go:1]: If you'd like help making this change, feel free to DM me and I'll help out with that.

**Pull request merged**